File: JAVA PROGRAMMING/Test67.java
// 67. FACTORS OF A NUMBER
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Num Value: ");
        int num = in.nextInt();

        for(int i = 1; i < num; i++) {
            if(num % i == 0) {
                System.out.print(i+" ");
            }
        }
        in.close();
    }
}

File: TYPESCRIPT PROGRAMMING/Test67.ts
// 67. FACTORS OF A NUMBER
const Factors = (num: number): void => {
    for(let i: number = 1; i <= num; i++) {
        if(num % i === 0) {
            process.stdout.write(`${i} `);
        }
    }
    console.log();
}
Factors(14);
Factors(55);